R/helpers.R

```diff
@@ -101,7 +83,7 @@
 
   commands <-
     c(commands,
-      dockerRunCommand(containerImage, rCommand, taskId))
+      dockerRunCommand(containerImage, rCommand, paste0("task-", taskId)))
```

Comment:
do we still need to prepend "task-" to the command line here?

Reply:
Docker run has a character limit that requires the container name to be at least 3 characters or more

Reply:
huh.. interesting. OK, seems fine then.

Reply:
Removed the name of the container